Utilities/CodeGeneration/EdFi.Ods.CodeGen/Generators/EntityOrmMappings.cs

```diff
@@ -23,13 +23,13 @@ namespace EdFi.Ods.CodeGen.Generators
 {
     public class EntityOrmMappings : GeneratorBase
     {
-        private readonly IViewsProvider _viewsProvider;
+        private readonly IAuthorizationDatabaseTableViewsProvider _viewsProvider;
```

Comment:
Same here, rename the field and constructor param.

Reply:
I fixed it .Thanks